packages/playground/windows/PlaygroundNativeModules/GridViewManager.cpp

```diff
@@ -104,4 +112,19 @@ void GridViewManager::UpdateProperties(
   }
 }
 
+void GridViewManager::CalculateLayoutOnChildren(xaml::FrameworkElement const &view) noexcept {
+  const auto grid = view.as<xaml::Controls::Grid>();
+  for (const auto gridChild : grid.Children()) {
+    const auto gridItem = gridChild.as<xaml::Controls::Grid>();
```

Comment:
Why is it safe to assume all of the GridViewManager's Grid's children are also Grids?

Reply:
Also assumes only one child. So... no idea how this VM is set up 😅

Reply:
Also, doesn't seem like you're making any Grid assumptions in what props/methods your calling. What base works better? FrameworkElement?

Reply:
Yeah, basically only GridItemViewManager instances should be children in this example (which are known to be Grid), but it doesn't have to be the case. Grid is just a simple example here where we want to apply things like Grid.Column / Grid.Row props to allow a level of XAML layout. Happy to change this at some point.